TaskSharper.Calender.WPF/ViewModels/TodayViewModels/CalendarEventShowDetailsViewModel.cs

```diff
@@ -41,6 +69,13 @@ public void OnNavigatedTo(NavigationContext navigationContext)
             var id = navigationContext.Parameters["id"].ToString();
 
             SelectedEvent = _calendarService.GetEvent(id);
+
+            EventIsTypeTask = SelectedEvent.Type == Event.EventType.Task;
+            EventIsTypeAppointment = SelectedEvent.Type == Event.EventType.Appointment;
+            EventIsStatusConfirmed = SelectedEvent.Status == Event.EventStatus.Confirmed ||
+                                     SelectedEvent.Status == Event.EventStatus.Completed;
```

Comment:
Completed is not the same as Confirmed. We don't really use "Completed", because it removes the event from the calendar.

Reply:
Fixed